cleanlab/object_detection/filter.py

```diff
@@ -181,7 +196,7 @@ def _find_label_issues(
 
 
 def _find_label_issues_per_box(
-    scores_per_box: List[np.ndarray], threshold: float
+    scores_per_box: List[np.ndarray], thr_classes, threshold_factor=1.0
```

Comment:
Update docstring to reflect change

Reply:
each bounding box in the `n-th` example and a threshold factor. Each box below or equal to [fill in] will be marked as an issue.

Reply:
https://github.com/cleanlab/cleanlab/blob/3024fad44f236c2f546610be35c3cfa5d011ce31/cleanlab/object_detection/filter.py#L201